docker/Dockerfile.arm

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.10-alpine AS builder
```

Comment:
This Dockerfile does not make sense, because it is identical to existing one - just ARGs are passed.

Please change Makefile to pass ARCH args that are set into existing Dockerfile. I.e. use the same Dockerfile for both architectures.

Reply:
Dockerfile.arm deleted, in Makefile I made `make docker_arm32v7` that pass this args